lib_standard_app/io.c

```diff
@@ -58,9 +58,7 @@ WEAK uint8_t io_event(uint8_t channel)
 
     switch (G_io_seproxyhal_spi_buffer[0]) {
         case SEPROXYHAL_TAG_BUTTON_PUSH_EVENT:
-#ifdef HAVE_BAGL
             UX_BUTTON_PUSH_EVENT(G_io_seproxyhal_spi_buffer);
```

Comment:
i would have expected us to use the ! HAVE_BAGL in the standard app
I'm probably missing something 🤔

Reply:
I simplified to only support the LNS case which events are handled the same if we use BAGL or NBGL.
It's a divergence with the version in ledger-secure-sdk master branch, but to keep it compatible, I would have to add so many #ifdef TARGET_NANOS that it won't be better IMO.